src/deploy/standalone/noobaa_syslog.conf

```diff
@@ -29,8 +40,8 @@ template(name="LogseneFormat" type="list" option.json="on") {
 }
 
 # For servers
-local0.*        /var/log/noobaa.log;RSYSLOG_FileFormat
-&stop
+# local0.*        /var/log/noobaa.log;RSYSLOG_FileFormat
```

Comment:
this file is getting full of commented out lines and things are unordered and messy. rsyslog.conf depends on order of statements, so I suggest that you move the outchannel definitions to this section where we actually define the log output for each facility rather than having it spread around the file.

Reply:
can you figure out why?